commands to create job and cronjob

ramnar opened the pull request:
Command to create job and cronjob

tengqm commented on content/en/docs/reference/kubectl/cheatsheet.md:
```diff
@@ -85,12 +85,14 @@ Kubernetes manifests can be defined in YAML or JSON. The file extension `.yaml`,
+kubectl apply -f ./dir                                                                            # create resource(s) in all manifest files in dir
+kubectl apply -f https://git.io/vPieo                                                             # create resource(s) from url
+kubectl create deployment nginx --image=nginx                                                     # start a single instance of nginx
+kubectl create job hello --image=busybox -- echo "Hello World"                                    # create a job which prints "hello world"
+kubectl create cronjob hello --image=busybox --schedule="*/1 * * * *" -- echo "Hello World"       # create a cron job which prints "hello world" for every minute
```
Thanks for adding this hints. I'd suggest we put the new lines this way:

```
kubectl create job hello \                     # create a job which prints "hello world"
   --image=busybox -- echo "Hello World"                                    
```

The intent is to avoid horizontal scrolling of the code segment considering that readers may have a screen not so big.

ramnar replied:
@tengqm - Good feedback. I have made the change. Please review

sftim reviewed (changes requested):
POSIX shells treat the comment (following the backslash and space) as part of the command line to `kubectl`. I'm afraid this proposed change is not valid from a technical perspective.

ramnar commented:
> POSIX shells treat the comment (following the backslash and space) as part of the command line to `kubectl`. I'm afraid this proposed change is not valid from a technical perspective.

@sftim - Thanks for the feedback. I have removed backslash and updated the command. Now it is similar to kubectl run command in the 'Interacting with running pods' section. Please review

ramnar commented:
> > POSIX shells treat the comment (following the backslash and space) as part of the command line to `kubectl`. I'm afraid this proposed change is not valid from a technical perspective.
> 
> @sftim - Thanks for the feedback. I have removed backslash and updated the command. Now it is similar to kubectl run command in the 'Interacting with running pods' section. Please review

@sftim , @tengqm  - Can you please review this and approve

tengqm commented on content/en/docs/reference/kubectl/cheatsheet.md:
```diff
@@ -90,6 +90,10 @@ kubectl apply -f ./my1.yaml -f ./my2.yaml      # create from multiple files
+kubectl create job hello                       # create a job which prints "hello world"
+--image=busybox -- echo "Hello World"                                    
+kubectl create cronjob hello --image=busybox   # create a cron job which prints "hello world" for every minute
+--schedule="*/1 * * * *" -- echo "Hello World"       
```
nits:  please consider indent line 93 and 96 by 2+ spaces so users can easily see those lines are not standalone commands;

tengqm commented on content/en/docs/reference/kubectl/cheatsheet.md:
```diff
@@ -90,6 +90,10 @@ kubectl apply -f ./my1.yaml -f ./my2.yaml      # create from multiple files
+kubectl create job hello                       # create a job which prints "hello world"
+--image=busybox -- echo "Hello World"                                    
+kubectl create cronjob hello --image=busybox   # create a cron job which prints "hello world" for every minute
+--schedule="*/1 * * * *" -- echo "Hello World"       
```
@sftim does have some valid concerns. So maybe we should use the format here:

https://github.com/kubernetes/website/pull/24701/files#diff-9b5338cfe4b946f1b17ff38106ad8410dc6687e17e8327d3fc4864a5ee8a48c5R71-R73

tengqm replied:
about line 96 and 97, you can either do 

```
kubectl create cronjob hello --image=busybox --schedule="*/1 * * * *" -- echo "Hello World"
```

or 

```
kubectl create cronjob hello --image=busybox \
  --schedule="*/1 * * * *" -- echo "Hello World"
```

The current format is confusing. It looks like two commands.

ramnar commented:
@tengqm - please check now

tengqm commented:
/lgtm

sftim commented on content/en/docs/reference/kubectl/cheatsheet.md:
```diff
@@ -90,6 +90,13 @@ kubectl apply -f ./my1.yaml -f ./my2.yaml      # create from multiple files
 kubectl apply -f https://git.io/vPieo          # create resource(s) from url
 kubectl create deployment nginx --image=nginx  # start a single instance of nginx
+
+# create a job which prints "hello world"
```
```suggestion
# create a Job which prints "Hello World"
```

ramnar replied:
@sfitm - done this change

sftim commented on content/en/docs/reference/kubectl/cheatsheet.md:
```diff
@@ -90,6 +90,13 @@ kubectl apply -f ./my1.yaml -f ./my2.yaml      # create from multiple files
+# create a job which prints "hello world"
+kubectl create job hello --image=busybox -- echo "Hello World" 
+
+# create a cronjob which prints "hello world" for every minute
```
```suggestion
# create a CronJon that prints "Hello World" every minute
```

ramnar replied:
@sftim - done this change

ramnar replied:
/hold

sftim replied:
> /hold

?
(I'm not sure why to hold)

sftim reviewed (commented):
@ramnar, would you be willing to revise this per the suggestions in this review?

sftim reviewed (commented):
Thanks @ramnar
/lgtm

ramnar commented:
/hold cancel

ramnar commented:
@krousey can you approve

kbhawkey commented:
Page preview:
https://deploy-preview-24701--kubernetes-io-master-staging.netlify.app/docs/reference/kubectl/cheatsheet/

kbhawkey commented:
/approve
